exchange/exchange_test.go

```diff
@@ -2215,15 +2216,31 @@ func runSpec(t *testing.T, filename string, spec *exchangeSpec) {
 		debugLog.Regexp = regexp.MustCompile(`[<>]`)
 	}
 
+	// Passthrough JSON Testing
+	var passthrough json.RawMessage
+	var err error
+	impExtInfoMap := make(map[string]ImpExtInfo)
+	if spec.PassthroughFlag {
+		passthrough, err = getPassthrough(&openrtb_ext.RequestWrapper{BidRequest: &spec.IncomingRequest.OrtbRequest})
+		if err != nil {
+			t.Errorf("%s: Exchange returned an unexpected error. Got %s", filename, err.Error())
+		}
+		impExtInfoMap["my-imp-id"] = ImpExtInfo{Passthrough: passthrough}
```

Comment:
I think we should shy away from introducing hidden requirements in our test framework to get a particular piece of our test framework to function. As a result, in order to make the pass through feature more dynamic, I suggest removing the `"my-imp-id"` dependency for the pass through feature and instead read the imp id from `spec.IncomingRequest.OrtbRequest` and use that as the index into `impExtInfoMap`.